accProcess.py

```diff
@@ -158,6 +158,10 @@ def main():
                             metavar='True/False', default=False, type=str2bool,
                             help="""Save intensity distribution
                              (default : %(default)s)""")
+    parser.add_argument('--intensityDistributionVals',
+                             metavar = "values of intensity distribution", default= "accelerometer/intensityVals.txt", type=str, nargs ='*',
```

Comment:
In Python it's best practice not to use spaces around `=` when it's a keyword/argument (though I think in R is the opposite?). 
https://www.flake8rules.com/rules/E251.html
Note this is only for keyword/arguments. A normal assignment elsewhere does have spaces. If you have a linter  installed (e.g. [flake8](https://anaconda.org/anaconda/flake8)) it will constantly bug you for these little pythonic things.